src/main/java/redis/clients/jedis/BinaryJedisCluster.java

```diff
@@ -28,11 +29,22 @@
     MultiKeyBinaryJedisClusterCommands, JedisClusterBinaryScriptingCommands, Closeable {
 
   public static final int HASHSLOTS = 16384;
-  protected static final int DEFAULT_TIMEOUT = 2000;
-  protected static final int DEFAULT_MAX_ATTEMPTS = 5;
+
+  /**
+   * Default timeout in milliseconds.
+   */
+  public static final int DEFAULT_TIMEOUT = 2000;
```

Comment:
`public` makes these reachable from `JedisClusterCommand.java` for its default timeout.